packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.2.0
```

Comment:
Can you please either update the `mainver` macro, or get rid of it (and `release` and `prever`)? ;-)

Reply:
For the record, I used the script. Sure I can do that.